routers/users.py

```
@@ -0,0 +1,60 @@
+            raise HTTPException(
+                status_code=status.HTTP_409_CONFLICT,
+                detail={"status": "error","data":{"message":"닉네임이 중복되었습니다."}}
+            )
+    new_user = {
+        "userId": str(len(users) + 1),
+        "email": data.email,
+        "name": data.name,
+        "password": get_password_hash(data.password),
+        "nickname": data.nickname,
+        "profile_image": data.profile_image,
+        "created_at": datetime.now().isoformat()
```

Ruff (0.14.11)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)